dojo/utils.py

```diff
@@ -328,7 +330,7 @@ def set_duplicate(new_finding, existing_finding):
         raise Exception("Can not add duplicate to itself")
     deduplicationLogger.debug('Setting new finding ' + str(new_finding.id) + ' as a duplicate of existing finding ' + str(existing_finding.id))
     if is_duplicate_reopen(new_finding, existing_finding):
-        set_duplicate_reopen_(new_finding, existing_finding)
+        set_duplicate_reopen(new_finding, existing_finding)
     new_finding.duplicate = True
     new_finding.active = False
     new_finding.verified = False
```

Comment:
prefer not to change this here as it's unrelated and there's another PR open touching this code probably leading to conflicts.

Reply:
Just to clarify, I didn't rename this.... `set_duplicate_reopen_` is undefined, so this code path is currently failing.